packages/block-editor/src/store/selectors.js

```diff
@@ -2374,7 +2367,6 @@ export const getAllowedBlocks = createRegistrySelector( ( select ) =>
 		},
 		( state, rootClientId ) => [
 			getBlockTypes(),
-			unlock( select( STORE_NAME ) ).getReusableBlocks(),
 			...getInsertBlockTypeDependants( select )( state, rootClientId ),
```

Comment:
Because of this dependent method, we'll have to keep using `createRegistrySelector`. See #66833.